### ISSUE TITLE
set-csonlinedialinconferencingtenantsettings needs update.

### ISSUE THREAD

**Issue by kenwith** [MicrosoftDocs/office-docs-powershell#207]
On this page there are two bits of text that contradict each other:
 
1. In the Description:

“The following parameters are not applicable to Skype for Business Online: AsJob, EnableDialOutJoinConfirmation, IncludeTollFreeNumberInMeetingInvites, MigrateServiceNumbersOnCrossForestMove, and **UseUniqueConferenceIds**”

2. In the Applies To for the parameter UseUniqueConferenceIds.

https://docs.microsoft.com/en-us/powershell/module/skype/set-csonlinedialinconferencingtenantsettings?view=skype-ps

**Comment by kenwith** [comment]
This might be a situation where [the tool](https://github.com/PowerShell/platyPS) that 'stubbed' out the content could 'see' the parameter. The PlatyPS tool sees whatever someone that runs Get-Command would see but parameters are not always applicable even though it shows as being 'seen' for that cmdlet. I will look into this with the PM and get this fixed asap. Also, just a note that the process is done in a fully transparent way and anyone could run PlatyPS and generate help content. It is super helpful when these issues are reported too so community can benefit and we can get these updated. Thank you Greig for reporting!

**Comment by kenwith** [comment]
All, this is a known bug in how I setup the source in GitHub. It has to do with what I mentioned above that the tool can 'see' that parameter but it is actually not applicable to Skype for Business Online. Trying to use that parameter would likely generate an error. I have this on my radar and the quick fix we came up with was to add that note in the description that the parameters listed are not applicable. I know this causes confusion because down in the details of the parameter it says "Applicable To" in the metadata for the parameter. That metadata was generated because the tool can see it so it thought it was applicable (but functionally it isn't) and thus we added that note. I will keep figuring out how to make this less confusing. Thanks for reporting.
